iroha_data_model/Cargo.toml

```diff
@@ -23,6 +23,7 @@ iroha_derive = { version = "=0.1.0", path = "../iroha_macro/iroha_derive" }
 iroha_macro = { version = "=0.1.0", path = "../iroha_macro" }
 iroha_version = { version = "=0.1.0", path = "../iroha_version" }
 iroha_error = { version = "=0.1.0", path = "../iroha_error" }
+iroha_http_server = { version = "=0.1.0", path = "../iroha_http_server" }
```

Comment:
The data model should not depend on `iroha_http_server` to my mind. For example there is a situation where `iroha_client` uses `iroha_data_model` and it will also load the whole http server then.

Maybe add a feature `http_server` here.

Another option to deal with this can be separating common http primitives and trait into an`iroha_http` crate. And here depend on `iroha_http`.